cmd/doc/cmd.go

```diff
@@ -0,0 +1,54 @@
+package doc
```

Comment:
Shouldn't this filename be `main.go`?

Reply:
Not necessarily. This is being called in cmd/pessimism/main.go. This file just implements the logic to run the doc generation